internal/schemautil/service.go

```diff
@@ -16,6 +16,9 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+// DefaultTimeoutMinutes is the default timeout for service operations.
+const DefaultTimeoutMinutes = 20
```

Comment:
By the var name I would naturally expect it to be of type `time` with some timeout

Reply:
this has been changed completely

Reply:
please check it again, I moved it to a function completely, also I checked with other open source projects, most use `Minutes` suffix when denoting an integer constant